packages/mds-repository/repository.ts

```diff
@@ -31,8 +35,10 @@ export const CreateRepository = <TRepositoryMethods>(
   methods: (connect: (mode: ConnectionMode) => Promise<Connection>) => TRepositoryMethods,
   { entities = [], migrations = [] }: RepositoryOptions = {}
 ) => {
+  // eslint-disable-next-line no-console
+  console.warn('CreateRepository is deprecated. Extend ReadWriteRepository instead.')
```

Comment:
I suppose this is to be replaced with instantiating an instance of the class now?

Reply:
Indeed. Right now, the repositories export a singleton, but once we have an active service component, that likely news up an instance.